components/ScriptLibrary/src/LHVM.cpp

```diff
@@ -106,274 +59,1424 @@ struct imemstream: virtual membuf, std::istream
 	{
 	}
 };
-} // namespace
+
+LHVM::LHVM()
+{
+	_currentStack = &_mainStack;
+
+	_opcodesImpl[0] = &LHVM::Opcode00End;
+	_opcodesImpl[1] = &LHVM::Opcode01Jz;
+	_opcodesImpl[2] = &LHVM::Opcode02Push;
+	_opcodesImpl[3] = &LHVM::Opcode03Pop;
+	_opcodesImpl[4] = &LHVM::Opcode04Add;
+	_opcodesImpl[5] = &LHVM::Opcode05Sys;
+	_opcodesImpl[6] = &LHVM::Opcode06Sub;
+	_opcodesImpl[7] = &LHVM::Opcode07Neg;
+	_opcodesImpl[8] = &LHVM::Opcode08Mul;
+	_opcodesImpl[9] = &LHVM::Opcode09Div;
+
+	_opcodesImpl[10] = &LHVM::Opcode10Mod;
+	_opcodesImpl[11] = &LHVM::Opcode11Not;
+	_opcodesImpl[12] = &LHVM::Opcode12And;
+	_opcodesImpl[13] = &LHVM::Opcode13Or;
+	_opcodesImpl[14] = &LHVM::Opcode14Eq;
+	_opcodesImpl[15] = &LHVM::Opcode15Neq;
+	_opcodesImpl[16] = &LHVM::Opcode16Geq;
+	_opcodesImpl[17] = &LHVM::Opcode17Leq;
+	_opcodesImpl[18] = &LHVM::Opcode18Gt;
+	_opcodesImpl[19] = &LHVM::Opcode19Lt;
+	_opcodesImpl[20] = &LHVM::Opcode20Jmp;
+	_opcodesImpl[21] = &LHVM::Opcode21Sleep;
+	_opcodesImpl[22] = &LHVM::Opcode22Except;
+	_opcodesImpl[23] = &LHVM::Opcode23Cast;
+	_opcodesImpl[24] = &LHVM::Opcode24Call;
+	_opcodesImpl[25] = &LHVM::Opcode25EndExcept;
+	_opcodesImpl[26] = &LHVM::Opcode26RetExcept;
+	_opcodesImpl[27] = &LHVM::Opcode27IterExcept;
+	_opcodesImpl[28] = &LHVM::Opcode28BrkExcept;
+	_opcodesImpl[29] = &LHVM::Opcode29Swap;
+	_opcodesImpl[30] = &LHVM::Opcode30Line;
+}
 
 /// Error handling
 void LHVM::Fail(const std::string& msg)
 {
-	throw std::runtime_error("LLVM Error: " + msg + "\nFilename: " + _filename.string());
+	throw std::runtime_error("LHVM Error: " + msg);
```

Comment:
Removing the filename from the error make me think that we can do without throwing exceptions. Instead return codes with string details like prototyped in #774